packages/component/src/Styles/StyleSet/SendBoxTextarea.js

```diff
@@ -0,0 +1,51 @@
+export default function createSendBoxTextareaStyle({
+  paddingRegular,
+  primaryFont,
+  sendBoxMaxHeight,
+  sendBoxTextColor,
+}) {
+  return {
+    alignItems: 'center',
+    fontFamily: primaryFont,
+    paddingBottom: paddingRegular,
+    paddingLeft: paddingRegular,
+    paddingRight: paddingRegular,
+    paddingTop: paddingRegular,
+
+    '& > div': {
+      fontFamily: 'inherit',
+      fontSize: 'inherit',
+      height: '100%',
+      maxHeight: sendBoxMaxHeight,
+      paddingBottom: 0,
```

Comment:
Why import `paddingBottom` etc if you're not using those variables?

Reply:
I copied and pasted the css from the textarea. Don’t actually think I need paddingBottom in the div styling.